### PR TITLE
[IMP] account_credit_control: UX improvements

### PR DESCRIPTION
**Pull request by carlosdauden**
WIP

@Tecnativa

### CODE REVIEW

File: account_credit_control/models/credit_control_run.py [hunk 1]
```diff
@@ -142,7 +158,11 @@ def generate_credit_lines(self):
         self._generate_credit_lines()
         return True
 
-    @api.multi
+    def unlink(self):
+        # Ondelete cascade don't check unlink lines restriction
```
Comment (pedrobaeza) [inline account_credit_control/models/credit_control_run.py:173]: I don't get why is this needed. You should put `ondelete="cascade"` on child many2one, not on the one2many.

File: account_credit_control/models/res_partner.py [hunk 2]
```diff
@@ -49,6 +49,19 @@ class ResPartner(models.Model):
...
+        fetch_data = self.env['credit.control.line'].read_group(
+            domain=[('partner_id', 'in', partners.ids)],
+            fields=['partner_id'],
+            groupby=['partner_id'],
+        )
+        result = {data['partner_id'][0]: data['partner_id_count']
+                  for data in fetch_data}
+        for partner in self:
+            partner.credit_control_count = result.get(partner.id, 0)
+        print(len(partners))
```
Comment (pedrobaeza) [inline account_credit_control/models/res_partner.py]: Remove this

File: account_credit_control/models/credit_control_run.py (hunk 1 above)
Comment (carlosdauden) [reply account_credit_control/models/credit_control_run.py:173]: I want to check this: https://github.com/OCA/account-financial-tools/pull/695/files#diff-de576053687d9842c8f5b25c790023fbR201

**Comment by pedrobaeza** [comment]
Is this finished?

**Comment by pedrobaeza** [comment]
@carlosdauden is this finished?

**Comment by carlosdauden** [comment]
Tomorrow

**Comment by pedrobaeza** [comment]
@carlosdauden friendly reminder

**Comment by pedrobaeza** [comment]
FYI, I have translated the module in current state in Weblate.

**Comment by carlosdauden** [comment]
Remove WIP

**Review by pedrobaeza — APPROVED** [review]
Let's finish this for v12 migration
